events/wc_undead_events.txt

```diff
@@ -1345,8 +1441,75 @@ character_event = {
 	show_from_from = yes
 
 	option = {
-		name = EVTOPTA_WCUND_355
+		trigger = {
+			NOT = {
+				religion = throsic
+			}
+		}
+		name = EVTOPTA_WCUND_355_abomination
+
+		custom_tooltip = { text = EVTOPTA_WCUND_355_tooltip }
+		event_target:target_prisoner = {
+			random_list = {
+				10 = {
+					modifier = {
+						factor = 1.5
+						trait = genius
+					}
+					modifier = {
+						factor = 1.3
+						trait = quick
+					}
+					modifier = {
+						factor = 1.3
+						trait = shrewd
+					}
+				}
+				40 = {
+					tooltip = { add_trait = slow }
+					hidden_effect = { ROOT = { set_character_flag = slow_mn_flag } }
+
+					modifier = {
+						factor = 1.5
+						trait = imbecile
+					}
+					modifier = {
+						factor = 1.3
+						trait = slow
+					}
+					modifier = {
+						factor = 1.3
+						trait = dull
+					}
+				}
+				50 = {
+					tooltip = { add_trait = imbecile }
+					hidden_effect = { ROOT = { set_character_flag = imbecile_mn_flag } }
+
+					modifier = {
+						factor = 1.5
+						trait = imbecile
+					}
+					modifier = {
+						factor = 1.3
+						trait = slow
+					}
+					modifier = {
+						factor = 1.3
+						trait = dull
+					}
+				}
+			}
+		}
+	}
+	
+	option = {
+		trigger = {
+			religion = throsic
+		}
+		name = EVTOPTA_WCUND_355_wickergolem
```

Comment:
Don't see the point of the new option. You could just relocalize the name.
https://ck2.paradoxwikis.com/Event_modding#Option_Name